Emby.Server.Implementations/TV/TVSeriesManager.cs

```diff
@@ -146,23 +146,10 @@ public IEnumerable<Episode> GetNextUpEpisodes(NextUpQuery request, User user, IE
             var allNextUp = seriesKeys
                 .Select(i => GetNextUp(i, currentUser, dtoOptions));
 
-            // allNextUp = allNextUp.OrderByDescending(i => i.Item1);
-
-            // If viewing all next up for all series, remove first episodes
-            // But if that returns empty, keep those first episodes (avoid completely empty view)
-            var alwaysEnableFirstEpisode = !string.IsNullOrEmpty(request.SeriesId);
-            var anyFound = false;
-
             return allNextUp
                 .Where(i =>
                 {
-                    if (alwaysEnableFirstEpisode || i.Item1 != DateTime.MinValue)
-                    {
-                        anyFound = true;
-                        return true;
-                    }
-
-                    if (!anyFound && i.Item1 == DateTime.MinValue)
+                    if (i.Item1 != DateTime.MinValue)
```

Comment:
You have to fix the remaining lines yourself. Can't do it from the app.
```suggestion
                    return i.Item1 != DateTime.MinValue;
```